exporter/elasticsearchexporter/README.md

```diff
@@ -81,6 +81,32 @@ All other defaults are as defined by [confighttp].
 
 The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. However, the sending queue is currently disabled by default.
 
+### Batching
+
+> [!WARNING]
+> The `batcher` config is experimental and may change without notice.
+
+The Elasticsearch exporter supports the [common `batcher` settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterbatcher/config.go).
+The `batcher` config is currently disabled by default.
```

Comment:
A little bit of ambiguity here but I don't know how to make it clearer. Strictly speaking, it is unset by default.

Reply:
I've reworded the paragraph below to make it clearer that the exporter does its own buffering/batching by default, let me know if it looks good to you.